com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDAdditionalCameraData.cs

```diff
@@ -37,13 +37,24 @@ public class HDPhysicalCamera
         // Lens
         // Note: focalLength is already defined in the regular camera component
         [SerializeField][Range(kMinAperture, kMaxAperture)] float m_Aperture = 16f;
+        [SerializeField][Min(0.1f)] float m_FocusDistance = 10f;
 
         // Aperture shape
         [SerializeField][Range(kMinBladeCount, kMaxBladeCount)] int m_BladeCount = 5;
         [SerializeField] Vector2 m_Curvature = new Vector2(2f, 11f);
         [SerializeField][Range(0f, 1f)] float m_BarrelClipping = 0.25f;
         [SerializeField][Range(-1f, 1f)] float m_Anamorphism = 0f;
 
+
+        /// <summary>
+        /// The focus distance of the lens, used in Depth of Field computations when the DoF volume is configured to read this value from the camera.
```

Comment:
The focus distance of the lens. The Depth of Field Volume override uses this value if you set `focusDistanceMode` to `FocusDistanceMode.Camera`.